pkg/controller/machine/machine_controller_suite_test.go

```diff
@@ -18,53 +18,94 @@ package machine
 
 import (
 	"context"
+	"flag"
 	"log"
-	"os"
 	"path/filepath"
 	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
 
 	machinev1 "github.com/openshift/api/machine/v1beta1"
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
+	"k8s.io/klog/v2/textlogger"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/envtest"
-	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+func init() {
+	textLoggerConfig := textlogger.NewConfig()
+	textLoggerConfig.AddFlags(flag.CommandLine)
+	logf.SetLogger(textlogger.NewLogger(textLoggerConfig))
+
+	// Register required object kinds with global scheme.
+	_ = machinev1.Install(scheme.Scheme)
```

Comment:
Why not do this in the BeforeSuite and check the error?

Reply:
This is copypasta from the machineset controller - but yes, that makes more sense!

Reply:
Ah - it's so the non ginkgo tests don't fail :/

Reply:
In that case, we can fix this up at some other point

Reply:
Have added `TestMain` with error checking and not moved to `BeforeEach()`